src/App.jsx

```diff
@@ -14,57 +17,57 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  let [value, setValue] = useState('Jam');
```

Comment:
Use "const"!
And use context named variables.
`const [selectedGood, setSelectedGood] = useState('Jam');`